jupyterlab_gitsync/setup.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from setuptools import find_packages, setup
+
+with open("README.md") as f:
+  long_description = f.read()
+
+version = None
+with open(os.path.join(os.getcwd(), "jupyterlab_gitsync",
+                       "version.py")) as f:
+  for l in f:
+    if l.startswith("VERSION"):
+      version = l.rstrip().split(" = ")[1].replace("'", "")
+
+if not version:
+  raise RuntimeError("Unable to determine version")
+
+npm_package = "jupyterlab_gitsync-{}.tgz".format(version)
+if not os.path.exists(os.path.join(os.getcwd(), npm_package)):
+  raise FileNotFoundError("Cannot find NPM package. Did you run `npm pack`?")
+
+data_files = [
+    ("share/jupyter/lab/extensions", (npm_package,)),
+    ("etc/jupyter/jupyter_notebook_config.d",
+     ("jupyter-config/jupyter_notebook_config.d/jupyterlab_gitsync.json",
+     )),
```

Comment:
If we're going to bundle `git-sync-changes` in the repository (rather than requiring the user to set it up ahead of time), then it should be included here as one of the data files.

Reply:
I think we should do the bundle (without copy paste the file into this repo)

Reply:
Will be added in future PR.